src/assets/createTokenBg.svg

```diff
@@ -0,0 +1,1431 @@
+<svg width="624" height="161" viewBox="0 0 624 161" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
```suggestion
<svg viewBox="0 0 624 161" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Check [this](https://github.com/aeternity/superhero-ui/pull/897#discussion_r556475843) discussion.